tag_pos:
  [{id: 0,x: 13.364,y: 22.551,z: 0.370, ex: 1.5708,ey: 0.0,ez: 3.1416},
  {id: 1,x: 1.395,y: 33.835,z: 0.370, ex: 1.5708,ey: 0.0,ez: 0.0},
  {id: 2,x: 13.120,y: 34.055,z: 0.370, ex: 1.5708,ey: 0.0,ez: -1.5708},
  {id: 3,x: 1.594,y: 22.358,z: 0.370, ex: 1.5708,ey: 0.0,ez: 1.5708}
  ]

